### DISCUSSION TITLE
Menu layout for individual sessions

### DISCUSSION THREAD

**Discussion by cociweb** [music-assistant/support#2651] [Feature requests and ideas]
Hello,

Currently, the menu can be set to either horizontal or vertical orientation. In the settings, you can choose your preference at the device level, which is fine and convenient.
I use a vertical menu for both HA and the individual MA on the same device / same browser

The idea occurred to me to embed MA into HA as a frame, and it works well. However, once I include MA in a frame, the vertical MA menu becomes inconvenient to use, so I opt for the horizontal menu. Then I realize that it's dependent on the device, so I'm in a constant battle with myself over which to choose.

Is there a way to adjust this behavior slightly? For example, checking a special header in HA, or something similar?

**Comment by OzGav** [comment]
This has been overtaken by changes in the UI and the bottom menu is no longer an option.
